pkg/reconciler/common/releases_test.go

```diff
@@ -731,59 +859,94 @@ func TestTargetManifest(t *testing.T) {
 				},
 			},
 		},
-		expectedNumResources: 2,
-		expectedError:        nil,
+		expectedManifestsPath: "",
+	}}
```

Comment:
Can you add a test case which verifies that multiple manifest paths in `AdditionalManifests`?

Reply:
done